variables.tf

```diff
@@ -18,6 +18,7 @@ variable volume_size {
 
 variable ssh_private_key {
   description = "Used to connect to the instance once created"
```

Comment:
As we are changing the interface, let's at least improve this description

Reply:
Better now?

Reply:
Maybe a little too much on the verbose side now. IMHO "The private key material of the key pair associated with the instance"

Reply:
Changed